src/main/io/ledstrip.c

```diff
@@ -741,6 +742,16 @@ void updateLedStrip(void)
         return;
     }
 
+    if ( IS_RC_MODE_ACTIVE(BOXLEDLOW)){
+        if (ledStripIsOnFlag){
+    		ledStripDisable();
+            ledStripIsOnFlag = false;
+        }
+        return;
+    }else{
+        ledStripIsOnFlag = true;
+    }
+
```

Comment:
there need to be a check here, something like:
```
if (!ledStripIsOnFlag) {
return;
}
```
right?

Perhaps also rename `ledStripIsOnFlag` to be simply `ledStripEnabled`?